### DISCUSSION TITLE
Parameter learn-ansible

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1599] [Q&A] [answered]
![Screenshot 2023-11-13 155941](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/42061300-48d1-4c70-a40b-a97be5bb5e36)
![Screenshot 2023-11-13 160008](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/e2c216e8-af09-4fb2-abd7-7a71349d94a3)
aws ssm get parameter --names "sample" this is ok
cd learn-ansible/  I'm unable find where i have done mistake. 
 I'm more confusing about inttelij. 
Any detailed Video about Inttelij please refer me.

**Comment by sreedharm07** [comment, marked as answer]
Hi @palsachennaiah 
i remember that you said in lab quarry session that you started directly from ansible. and suggested to start from 1 st class, did you started. 
i think you not started, becasue the error you get now was a very basic and that was teached in very starting sessions. please start from starting sessions for a better understanding. 

Anyway the error was you dint cloned any repos in your workstation. if there is no repos cloned in your machine then how can we do cd to that repo ?
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/508cebc1-f610-4196-bd7f-b5084adfdd5b)
